dsa-23au/java-dsa/datasets/src/test/java/dev/codewithfriends/NameCrawlerTest.java

```diff
@@ -0,0 +1,24 @@
+package dev.codewithfriends;
```

Comment:
Overall with the project, what is your main goal with the name database?

Reply:
The goal is to crawl a large, interesting dataset that we can use to profile, compare, and measure the performance of different data structures. Perhaps even have a name recommendation service: given some qualities you want to convey or imbue, what are some suggested names?